server/re/src/irods_re_serialization.cpp

```diff
@@ -928,24 +963,32 @@ namespace irods::re_serialization
            serialized_parameter_t&  _out) {
         try {
             genQueryOut_t* tmp = boost::any_cast<genQueryOut_t*>(_p);
-            if (tmp) {
-                genQueryOut_t* l = tmp;
-
-                _out["rowCnt"] = boost::lexical_cast<std::string>(l->rowCnt);
-                _out["attriCnt"] = boost::lexical_cast<std::string>(l->attriCnt);
-                _out["continueInx"] = boost::lexical_cast<std::string>(l->continueInx);
-                _out["totalRowCount"] = boost::lexical_cast<std::string>(l->totalRowCount);
 
-                for (int i = 0; i < l->attriCnt; ++i) {
-                    for (int j = 0; j < l->rowCnt; ++j) {
+            if (tmp) {
+                _out["rowCnt"] = boost::lexical_cast<std::string>(tmp->rowCnt); // NOLINT(boost-use-to-string)
+                _out["attriCnt"] = boost::lexical_cast<std::string>(tmp->attriCnt); // NOLINT(boost-use-to-string)
+                _out["continueInx"] = boost::lexical_cast<std::string>(tmp->continueInx); // NOLINT(boost-use-to-string)
+                // NOLINTNEXTLINE(boost-use-to-string)
+                _out["totalRowCount"] = boost::lexical_cast<std::string>(tmp->totalRowCount);
+
+                for (int i = 0; i < tmp->attriCnt; ++i) {
+                    for (int j = 0; j < tmp->rowCnt; ++j) {
                         std::string i_str = boost::lexical_cast<std::string>(i);
                         std::string j_str = boost::lexical_cast<std::string>(j);
-                        _out["attriInx_" + i_str] = boost::lexical_cast<std::string>(l->sqlResult[i].attriInx);
-                        _out["len_" + i_str] = boost::lexical_cast<std::string>(l->sqlResult[i].len);
-                        _out["value_" + j_str + "_" + i_str] = boost::lexical_cast<std::string>(l->sqlResult[i].value+j*l->sqlResult[i].len);
+                        // NOLINTNEXTLINE(boost-use-to-string, cppcoreguidelines-pro-bounds-constant-array-index)
+                        _out["attriInx_" + i_str] = boost::lexical_cast<std::string>(tmp->sqlResult[i].attriInx);
+                        // NOLINTNEXTLINE(boost-use-to-string, cppcoreguidelines-pro-bounds-constant-array-index)
+                        _out["len_" + i_str] = boost::lexical_cast<std::string>(tmp->sqlResult[i].len);
+
+                        // clang-format off
+                        _out[fmt::format("value_{}_{}", j_str, i_str)] =
+                            // NOLINTNEXTLINE(bugprone-implicit-widening-of-multiplication-result, cppcoreguidelines-pro-bounds-constant-array-index, cppcoreguidelines-pro-bounds-pointer-arithmetic)
+                            boost::lexical_cast<std::string>(tmp->sqlResult[i].value + j * tmp->sqlResult[i].len);
```

Comment:
Capture the bugprone clang-tidy error in an issue.

Reply:
Is this done?

Reply:
The line in question will change with this PR. I'm going to create the issue after this is merged.

Reply:
Makes sense. So this will be resolved after merging, then.

Reply:
Yep

Reply:
Created #7222